src/diffusers/models/activations.py

```diff
@@ -146,6 +146,18 @@ def forward(self, hidden_states):
         return hidden_states * self.activation(gate)
 
 
+class SiLU(nn.Module):
```

Comment:
Does this have a different name? silu normally does not come with the linear projection
can be `HunyuanVideoSiLU` if we cannot come up with anything else

Reply:
yes, still looking for this one. It's not common and mostly swiglu is used instead

Reply:
I've created a `LinearActivation` class for cases like this where we use a normal linear with an activation. LMK your thoughts since this seemed like a neater way to do it now, and maybe for future models that follow this pattern. So as to not mess with existing naming, this layer can be set by specifying `linear-silu` instead of `silu` (which would be rather confusing)